roles/lnls-ans-role-servweb/README.md

```diff
@@ -0,0 +1,90 @@
+LNLS Ansible Role Servweb
+=========================
+
+This Ansible role clone control-system-constantsmachines.
```

Comment:
Small typo in `control-system-constantsmachines`

Reply:
deleted the whole role.